config.json

```diff
@@ -680,7 +680,13 @@
       "core": false,
       "difficulty": 6,
       "slug": "all-your-base",
-      "topics": null,
+      "topics": [
+        "loops",
+        "mathematics",
+        "integers",
+        "arrays",
+        "conditionals"
```

Comment:
I would also add `exception_handling` since the tests expect an exception to be thrown :)